### ISSUE TITLE
Does current libcompose support the docker-compose "deploy"?

### ISSUE THREAD

**Issue by rajeevb2** [docker/libcompose#463]
This feature is explained in:
https://sreeninet.wordpress.com/2017/03/28/comparing-docker-compose-versions/

**Comment by cdrage** [comment]
Nope. That's in v3! Which is currently in-progress of getting added.
